compiler/rustc_query_system/src/dep_graph/graph.rs

```diff
@@ -1098,7 +1098,7 @@ impl<D: Deps> CurrentDepGraph<D> {
         use std::time::{SystemTime, UNIX_EPOCH};
 
         let duration = SystemTime::now().duration_since(UNIX_EPOCH).unwrap();
-        let nanos = duration.as_secs() * 1_000_000_000 + duration.subsec_nanos() as u64;
+        let nanos: u64 = duration.as_nanos().try_into().unwrap();
```

Comment:
same as above w.r.t. u128

Reply:
Yes, there it looks ok to change to u128.